distrobox-init

```diff
@@ -1006,11 +1015,14 @@ eval ${init_hook}
 if [ "${init}" -eq 0 ]; then
 	printf "container_setup_done\n"
 	# Keepalive loop
-	HOST_WATCH="
-		/etc/hosts
-		/etc/localtime
-		/etc/resolv.conf
-	"
+	HOST_WATCH="/etc/localtime"
+	if [ "${custom_networking}" -eq 0 ]; then
```

Comment:
Technically we can leave these untouched
If it's a custom network, they will not be mountpoints, so the subsequent check will not pick them up for sync

Reply:
If I recall, I also expected they wouldn't be mount points but networking still broke for some reason. Should be a temporary workaround until that's looked into.